js/src/test/scala/com/thirdparty/defs/tags/MathMLTags.scala

```diff
@@ -0,0 +1,161 @@
+package com.thirdparty.defs.tags
+
+import com.thirdparty.tags.MathMlTag
+import org.scalajs.dom
+
+// #NOTE: GENERATED CODE
+//  - This file is generated at compile time from the data in Scala DOM Types
+//  - See `GeneratorSpec.scala` for code generation params
+//  - Contribute to https://github.com/raquo/scala-dom-types to add missing tags / attrs / props / etc.
+
+trait MathMLTags {
+
+
+  def mathMLTag[El <: dom.MathMLElement](name: String): MathMlTag[El] = new MathTag(name)
+
+
+  /**
+    * The top-level element in MathML, representing a mathematical expression.
+    *
+    * [[https://developer.mozilla.org/en-US/docs/Web/MathML/Element/math math @ MDN]]
+    */
+  lazy val math: MathMlTag[dom.MathMLElement] = mathMLTag("math")
+
+
+  /**
```

Comment:
The contents of this file don't match `MathMLTagDefs`, e.g. it does not have the `annotation` element – seems that you need to regenerate again.

Reply:
Regenerated... the CI now actually fails on my fork, which hI"m looking into because I don't understand why :-/

Reply:
```Run sbt "project domtypesJVM" test "project domtypesJS" +test
/home/runner/work/_temp/6b77182c-5dce-4bb9-b7f4-7b27e2ccac05.sh: line 1: sbt: command not found
```

I'm ... really not sure what's happened.